trace-dispatcher/src/Cardano/Logging/DocuGenerator.hs

```diff
@@ -428,8 +431,14 @@ docuResultsToText dt@DocTracer {..} configuration = do
       header4  = fromText "\n## Datapoints\n\n"
       contentD = mconcat $ intersperse (fromText "\n\n")
                               (map (unpackDocu . snd) datapointBuilders)
-      config  = fromString $ "\n\nConfiguration: " <> show configuration <> "\n\n"
+      config  = fromText $ "\n## Configuration: \n```\n"
+                            <> decodeUtf8 (BS.toStrict $
+                                            AE.encodePretty configuration)
+                            <> "\n```\n"
       numbers = fromString $  show (length dtBuilderList) <> " log messages." <> "\n\n"
+      legend  = fromString $  "\9443 - This is the root of a tracer\n\n"
+                           <> "\9442 - This is the root of a tracer that is silent because of the current configuration\n\n"
+                           <> "\9436 - This is the root of a tracer, that provides metrics\n\n"
```

Comment:
A general suggestion for these special characters: can we instead of inlining them, use a top-level definition (and use hex encoding for UTF codes), like so:
```
utf16CircledT :: Builder
utf16CircledT = singleton '\x24E3'
```
This would make any maintenance / change to the special chars much more easy.

Reply:
Done